source/_integrations/squeezebox.markdown

```diff
@@ -244,3 +242,12 @@ This action can be used to integrate a Squeezebox query into an automation. For
 `hass.services.call("squeezebox", "call_query", { "entity_id": "media_player.kitchen", "command": "albums", "parameters": ["0", "20", "search:beatles", "tags:al"] })`
 To work with the results:
 `result = hass.states.get("media_player.kitchen").attributes['query_result']`
+
+### Action `button`
+
+This simple action allows you to mimic "pressing" a button on a hardware Squeezebox player, such as a Squeezebox Radio. A number of standard buttons, such as Preset 0 and Brightness Up, are available in the dropdown, or you can enter your own button name. Further information can be found at `https://lyrion.org/reference/cli/players/#button`
+
+| Data attribute | Optional | Description                                                     |
```

Comment:
on second thought.. May I ask you to convert this table into a list? We are trying to move away from tables..
In the format shown here:
<https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/docs-actions>